src/core/linear_solver/src/preconditioner/4C_linear_solver_preconditioner_muelu.cpp

```diff
@@ -71,8 +74,7 @@ void Core::LinearSolver::MueLuPreconditioner::setup(bool create, Epetra_Operator
 
       Teuchos::ParameterList& inverseList = muelulist_.sublist("MueLu Parameters");
 
-      std::string xmlFileName = inverseList.get<std::string>("MUELU_XML_FILE");
-      if (xmlFileName == "none") FOUR_C_THROW("MUELU_XML_FILE parameter not set!");
+      auto xmlFileName = inverseList.get<std::string>("MUELU_XML_FILE");
```

Comment:
What is the difference here in comparison to `tekolist_.sublist("Teko Parameters").get<std::filesystem::path>("TEKO_XML_FILE");`? Shouldn't it be the same code?

Reply:
Initially, I tried to adapt the code in such a way that as little code as possible is changed. Looking at the diff now, I agree that this should be the same code everywhere. Ill see if I an unify that.